tests/config.bats

```diff
@@ -81,6 +81,19 @@ function check_matrix() {
   check_matrix 'Config.Entrypoint' '[/bin/sh -c /ENTRYPOINT]'
 }
 
+@test "config --unsetlabel" {
```

Comment:
Looks like it's depending on a previously-run test to have pulled the image.  Needs to `_prefetch` for `--pull=false`, no?

Reply:
Done prefetching needed base image